framework/test/e2e/spark-cicd-userdefined-pipeline.e2e.test.ts

```diff
@@ -0,0 +1,108 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * E2E test for SparkCICDPipeline
+ *
+ * @group e2e/processing/spark-cicd-userdefined
+ */
+
+import { RemovalPolicy, CfnOutput, Stack, StackProps, App } from 'aws-cdk-lib';
+import { Bucket } from 'aws-cdk-lib/aws-s3';
+import { CodePipelineSource } from 'aws-cdk-lib/pipelines';
+import { TestStack } from './test-stack';
+import { SparkEmrCICDPipeline, SparkImage } from '../../src/processing';
+import { ApplicationStackFactory, CICDStage } from '../../src/utils';
+
+jest.setTimeout(9000000);
+
+// GIVEN
+const app = new App();
+const cicdStack = new Stack(app, 'CICDStack', {
```

Comment:
The stack name must be different from the default CICD e2e test to avoid deployment error during full e2e test